portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -10,24 +10,14 @@ import cromwell.pipeline.datastorage.dto.auth.AccessTokenContent
 import org.slf4j.LoggerFactory
 
 import scala.concurrent.ExecutionContextExecutor
+import scala.util.{ Failure, Success, Try }
 
 object CromwellPipelineApp extends App {
 
-  implicit val system: ActorSystem = ActorSystem("cromwell-pipeline")
-  implicit val executionContext: ExecutionContextExecutor = system.dispatcher
-
   val log = LoggerFactory.getLogger(CromwellPipelineApp.getClass)
```

Comment:
It would be better if you keep the previous order

Reply:
Which one?

Reply:
For example variable `log` and implicites.
You replaced the lines in order, but that doesn't make any sense.

Reply:
Is it important to keep previous order? I've put log before implicits, bc I want them to stay together with rejectionHandler, but it's a big block of code and logger could be inconspicuous this way